### ISSUE TITLE
GpuShapeView broken

### ISSUE THREAD

**Issue by BioRyajenka** [korlibs/korge#1166]
Hello!

This commit (672d058b) seem to broke `gpuShapeView.stroke` somewhere. It draws red half-plane instead of stroke 
![image](https://user-images.githubusercontent.com/9904421/209591952-5c1be7ac-d989-4dc9-ac5a-fe2f9f53aa57.png)

**Comment by soywiz** [comment]
Yes. This is one of the regressions that happened during the huge AG rewrite. Let's fix it before the next alpha
